.github/workflows/api-ci-using-gradle.yml

```diff
@@ -37,6 +37,13 @@ jobs:
       with:
         dependency-graph: ${{ github.ref == 'refs/heads/main' && 'generate-and-submit' || 'generate' }}
 
+    - name: Setup Node.js 20.x
```

Comment:
- [ ] See if we can replace this with `download.set(true)` in https://github.com/beercan1989/oauth-api/pull/117/files#diff-d86e96d993c0e2755bb4acf118eb261d586d1b150d249d99c4b05698953511afR16 as including NPM is breaking the CodeQL integration for the "Analyze (Java)" job https://github.com/beercan1989/oauth-api/actions/runs/8021880500/job/21914973530?pr=117